Monitoring in a Box/Ansible/roles/mib_setup/tasks/main.yaml

```diff
@@ -22,5 +22,8 @@
 - name: Set up MiB service
   import_tasks: set_up_mib_service.yaml
 
+- name: Remove SSH authorized keys from MiB VM
```

Comment:
It should be "Add" and not "Remove" right?

Reply:
This task was previously created to remove all ssh pub keys in the VM environment and I forgot to change the name. Will edit it to "add end user ssh public key" instead